addons/web_tour/static/tests/tour_recorder.test.js

```diff
@@ -616,49 +425,3 @@ test("Edit input after autofocus", async () => {
     checkTourSteps([".o_input"]);
     expect(tourRecorder.state.steps.map((s) => s.run)).toEqual(["edit Bismillah"]);
 });
-
-test("'Disable Tours' clean the custom running tour", async () => {
```

Comment:
becomes irrelevant ?

Reply:
Yes, the tour and custom tour work with the same service, so no more tests that test copied features